cf_xarray/accessor.py

```diff
@@ -1010,15 +1010,19 @@ def __repr__(self):
         coords = self._obj.coords
         dims = self._obj.dims
 
-        def make_text_section(subtitle, vardict, valid_values, valid_keys=None):
+        def make_text_section(subtitle, vardict, valid_values, default_keys=None):
 
             star = " * "
             tab = len(star) * " "
             subtitle = f"- {subtitle}:"
 
-            # Sort keys: Valid keys + additional in alphabetical order
-            valid_keys = [] if not valid_keys else list(valid_keys)
-            ordered_keys = valid_keys + sorted(set(vardict) - set(valid_keys))
+            # Sort keys: Order alphabetically if extra keys are available,
+            # otherwise preserve default keys order
+            default_keys = [] if not default_keys else list(default_keys)
+            extra_keys = list(set(vardict) - set(default_keys))
+            ordered_keys = (
+                sorted(default_keys + extra_keys) if extra_keys else default_keys
```

Comment:
sorry for not being clear, how about `default_keys + sorted(extra_keys)` so there's some consistency.

Reply:
That would bring it back to the initial behavior I think. Missing keys are appended at the bottom.
Do you mean this?
```
area: n/a
volume: ['foo']
foo_measure: ['foo']
```
Would work well as long as we keep only 2 default measures, but with more `meas1, meas2, meas3: n/a` wouldn't work.